package.json

```diff
@@ -95,6 +95,7 @@
     "cypress": "^6.5.0",
     "cypress-localstorage-commands": "^1.4.0",
     "cypress-real-events": "^1.2.0",
+    "esbuild-loader": "^2.12.0",
```

Comment:
Looks like this is leftover from some earlier esbuild attempt.

Reply:
Yeah thanks